src/main/java/com/google/codeu/servlets/UserListServlet.java

```diff
@@ -1,23 +1,35 @@
 package com.google.codeu.servlets;
 
+import com.google.codeu.data.Datastore;
+import com.google.gson.Gson;
 import java.io.IOException;
 
+import java.util.Set;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.codeu.data.Datastore;
 
 /**
  * Handles fetching all users for the community page.
  */
 @WebServlet("/user-list")
 public class UserListServlet extends HttpServlet {
-  private Datastore datastore = new Datastore();
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException {
-    response.getOutputStream().println(datastore.getUsers().toString());
+    response.setContentType("application/json");
+    Set<String> users = datastore.getUsers();
+    Gson gson = new Gson();
+    String json = gson.toJson(users);
```

Comment:
I know this is how it was written in the guide (probably to get us familiar with gson/json), but I'm not a fan of this variable name. I guess it's not so crucial here but I think we should try to use clearer and more descriptive variable names in the future!